src/status_im2/contexts/share/view.cljs

```diff
@@ -0,0 +1,163 @@
+                              {:text-to-copy      profile-qr-url
+                               :post-copy-message (i18n/label :t/link-to-profile-copied)})
+          :on-long-press    #(copy-text-and-show-toast
+                              {:text-to-copy      profile-qr-url
+                               :post-copy-message (i18n/label :t/link-to-profile-copied)})}
+         [quo/text
+          {:style           (style/profile-address-content link-to-profile-max-width)
+           :size            :paragraph-1
+           :weight          :medium
+           :ellipsize-mode  :middle
+           :number-of-lines 1}
+          profile-qr-url]]]
+       [rn/view {:style style/share-button-container}
+        [quo/button
+         {:icon                true
+          :type                :blur-bg
+          :size                32
+          :accessibility-label :link-to-profile
+          :override-theme      :dark
+          :on-press            #(list-selection/open-share {:message profile-qr-url})}
+         :i/share]]]]
+
+     [rn/view {:style style/emoji-hash-container}
+      [rn/view {:style style/profile-address-container}
+       [rn/view {:style style/profile-address-column}
+        [quo/text
+         {:size   :paragraph-2
+          :weight :medium
+          :style  style/emoji-hash-label}
+         (i18n/label :t/emoji-hash)]
+        [rn/touchable-highlight
+         {:active-opacity   1
+          :underlay-color   colors/neutral-80-opa-1-blur
+          :background-color :transparent
+          :on-press         #(copy-text-and-show-toast
+                              {:text-to-copy      emoji-hash
+                               :post-copy-message (i18n/label :t/emoji-hash-copied)})
+          :on-long-press    #(copy-text-and-show-toast
+                              {:text-to-copy      emoji-hash
+                               :post-copy-message (i18n/label :t/emoji-hash-copied)})}
+         [rn/text {:style (style/emoji-hash-content emoji-hash-max-width)} emoji-hash]]]]
+      [rn/view {:style style/share-button-container}
+       [quo/button
+        {:icon                true
+         :type                :blur-bg
+         :size                32
+         :accessibility-label :link-to-profile
+         :override-theme      :dark
+         :style               {:margin-right 12}
+         :on-press            #(copy-text-and-show-toast
+                                {:text-to-copy      emoji-hash
+                                 :post-copy-message (i18n/label :t/emoji-hash-copied)})
+         :on-long-press       #(copy-text-and-show-toast
+                                {:text-to-copy      emoji-hash
+                                 :post-copy-message (i18n/label :t/emoji-hash-copied)})}
+        :i/copy]]]]))
+
+(defn wallet-tab
+  []
+  [:<>
+   [rn/text {:style {:color colors/white :text-align :center}} "not implemented"]])
+
+(defn view
+  []
+  (let [selected-tab (reagent/atom profile-tab-id)]
```

Comment:
This `selected-tab` binding is a bit suspicious. A correct form-2 component should initialize state outside the renderer function. It should probably be like the following, similar to `status-im2.contexts.chat.home.view`.

```clojure
(defn view
  []
  (let [selected-tab (reagent/atom profile-tab-id)]
    (fn []
      [safe-area/consumer
       (fn [{:keys [top bottom]}]
         ...)])))
```

Reply:
resolved here : https://github.com/status-im/status-mobile/pull/15528/commits/bcd8198f1ece5a75ad440c54b4e2f07ff372da1f